### ISSUE TITLE
Feature request: alerts only after x failed executions

### ISSUE THREAD

**Issue by BlackScreen** [healthchecks/healthchecks#715]
Hello,
I am using Healthchecks in conjunction with Runitor for my cron jobs that download JSON files from other APIs to a local cache.
Sometimes those download timeout. Currently I then get an email immediately after the first failure - which is good, but does clutter my inbox, since there are a few failures during the day.
It would be awesome to set a threshold to send the alerts via mail (or other types) only after x failures. For my case: setting this threshold to 2 would be enough to get much less mails.
Maybe you could consider this in an upcoming version? Would be awesome. Thanks! :)

**Comment by cuu508** [comment]
Duplicate of #525
